modules/programs/starship.nix

```diff
@@ -85,7 +85,11 @@ in
   };
 
   config = mkIf cfg.enable {
-    home.packages = [ cfg.package ];
+    home = {
+      packages = [ cfg.package ];
+
+      sessionVariables.STARSHIP_CONFIG = "${config.xdg.configHome}/starship.toml";
```

Comment:
Makes me wonder if it would be worth creating an option, similar to we did in [docker-cli](https://github.com/nix-community/home-manager/pull/7411/files#diff-4af3e473b841c3256ff98005885ee5658038a1fd6c77948370de1f102fdbcb7fR24-R30)